src/client/dfs/dfs.c

```diff
@@ -3327,10 +3327,11 @@ dfs_release(dfs_obj_t *obj)
 		rc = -DER_IO_INVAL;
 	}
 
-	if (rc)
+	if (rc) {
 		D_ERROR("Failed to close DFS object, "DF_RC"\n", DP_RC(rc));
-	else
+	} else {
 		D_FREE(obj);
+	}
```

Comment:
Why add brackets? I think the preferred style is to not add them if not needed

Reply:
@daltonbohning yeah I removed this in the next commit I pushed for check patch. Didn't mean to add these.